lib/stripe/stripe_configuration.rb

```diff
@@ -27,7 +27,6 @@ module Stripe
   class StripeConfiguration
     attr_accessor :api_key
     attr_accessor :api_version
-    attr_accessor :client_id
```

Comment:
This isn't technically accessible until https://github.com/stripe/stripe-ruby/pull/952/files is merged. I can just pull that PR into this one if preferred.

Reply:
Just merged #952.

Reply:
Thanks! I've just updated the branch.